cli/src/electron/add.ts

```diff
@@ -1,11 +1,18 @@
 import { exec } from 'child_process';
 import { Config } from '../config';
 import { copyTemplate, hasYarn, installDeps, runTask } from '../common';
+import { readFileSync, writeFileSync } from '../util/fs';
+import * as path from 'path';
```

Comment:
Since you are only using join, no need to import *
```suggestion
import { join } from 'path';
```